cdap-api/src/main/java/co/cask/cdap/api/lineage/Transform.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.lineage;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Represent the TRANSFORM operation from collection of fields to another collection.
+ */
+public class Transform extends Operation {
+  private final Set<String> input;
+  private final Set<String> output;
```

Comment:
Same here: "split-on-comma" would return two fields, one before and one after the comma. Order matters.

Reply:
Using List now.